frontend/app_flowy/packages/flowy_infra_ui/lib/src/flowy_overlay/overlay_container.dart

```diff
@@ -18,7 +18,7 @@ class OverlayContainer extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final theme = context.watch<AppTheme>();
+    final theme = context.watch<AppTheme?>() ?? AppTheme.fromType(ThemeType.light);
```

Comment:
Hardcoding a theme here might bring bugs in other parts of the apps.

Reply:
It is not hardcoded, I just set a default value so if the former comes as null it doesn't break the code.
The better way of doing this is passing the AppTheme from the Widget's constructor so it can be used anywhere without thinking about the provider's value above the widget tree

Reply:
Thank you for contributing. I think it makes sense.

Reply:
> It is not hardcoded, I just set a default value so if the former comes as null it doesn't break the code. The better way of doing this is passing the AppTheme from the Widget's constructor so it can be used anywhere without thinking about the provider's value above the widget tree

got it, Thanks for explaining.